sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.assessApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,146 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeApplication,
+  saveFakeApplicationOfferingRequestChange,
+} from "@sims/test-utils";
+import {
+  ApplicationOfferingChangeRequestStatus,
+  ApplicationStatus,
+} from "@sims/sims-db";
+
+describe("ApplicationOfferingChangeRequestAESTController(e2e)-assessApplicationOfferingChangeRequest", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should assess the application offering change request for the provided application offering change request id.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource, undefined, {
+      applicationStatus: ApplicationStatus.Completed,
+    });
+    const applicationOfferingChangeRequest =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          application,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          },
+        },
+      );
+    const note = "Some dummy note.";
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+    const endpoint = `/aest/application-offering-change-request/${applicationOfferingChangeRequest.id}`;
+    // Act/Assert
+    await request(app.getHttpServer())
+      .patch(endpoint)
+      .send({
+        note,
+        applicationOfferingChangeRequestStatus:
+          ApplicationOfferingChangeRequestStatus.Approved,
+      })
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+    // Check if the application offering change request was assessed as expected.
+    const updatedApplicationOfferingChangeRequest =
+      await db.applicationOfferingChangeRequest.findOne({
+        select: {
+          id: true,
+          applicationOfferingChangeRequestStatus: true,
+          assessedNote: { id: true, description: true },
+        },
+        relations: { assessedNote: true },
+        where: { id: applicationOfferingChangeRequest.id },
+      });
+    expect(
```

Comment:
Just added a similar comment to another PR, we should always try to validate all the impacted DB columns as possible.
For instance, the modifier can also be checked here and I missed on previous reviews.